tla-assignments/src/main/scala/at/forsyte/apalache/tla/assignments/passes/PrimingPassImpl.scala

```diff
@@ -19,14 +19,18 @@ import com.typesafe.scalalogging.LazyLogging
 class PrimingPassImpl @Inject()(options: PassOptions,
                                 tracker: TransformationTracker,
                                 @Named("AfterPriming") nextPass: Pass with TlaModuleMixin )
-      extends PrimingPass with LazyLogging {
+  extends PrimingPass with LazyLogging {
   /**
     * The name of the pass
     *
     * @return the name associated with the pass
     */
   override def name: String = "PrimingPass"
 
+  private def trSeq( seq: Seq[TlaExTransformation] ) : TlaExTransformation = {
```

Comment:
Assuming `tr` is short for `translate`, using the full word would help with readability imo.

Reply:
`tr` is standard shorthand for `transform`, as in apply a `TlaExTransformation`. But sure, easy to rename.

Reply:
Ah! Where is this standardized? I'm not familiar but would interesting to learn the context :)

Given the amount of meanings for `tr` recorded by [wiktionary](https://en.wiktionary.org/wiki/tr.), I think the disambiguation is probably worth a few extra characters :)

Reply:
Duh. nm here. I finally our point about the relation between `tr` and the type. Sorry for being dense here. I think that should suffice for someone reading more carefully than I have here :)